knexfile.js

```diff
@@ -32,5 +32,14 @@ module.exports = {
   development: defaults,
   dummy: Object.assign({}, defaults, { seeds: { directory: './seeds/dummy' } }),
   staging: defaults,
-  production: merge({connection: {ssl: true}}, defaults)
+  production: merge({connection: {ssl: true}}, defaults),
+  docker: Object.assign({},
```

Comment:
should this be part of this PR? Not sure what it's doing

Reply:
@richchurcher made this change to allow for easier migrations.

Reply:
cool